packages/model-viewer/src/features/scene-graph/model.ts

```diff
@@ -56,6 +61,19 @@ export class LazyLoader {
   }
 }
 
+/**
+ * Facades variant mapping data.
+ */
+export class VariantData {
+  name: string;
+  index: number;
+  materialVariants = new Array<number>();
```

Comment:
maybe just `materials`? These are just the material indices this variant points to,  right?

Reply:
Yes, removing this field, it's not being used anywhere.